assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -335,6 +335,12 @@ che.infra.kubernetes.password=
 che.infra.kubernetes.oauth_token=
 che.infra.kubernetes.trust_certs=
 
+# Defines the way how servers are exposed to the world in k8s infra.
+# List of  strategies implemented in Che: default-host, multi-host, single-host
+che.infra.kubernetes.server_strategy=default-host
```

Comment:
How is this related with `che.single.port`? Does this property makes sense if single port is set to `false`?

Reply:
It is not related to a single port AFAIK

Reply:
Ok just wondering how `multi-host` works with single port set to `false`. But in any case this is not related to this PR so ok to merge for me.

Reply:
AFAIK Single host mode is used by docker infrastructure only. So neither k8s nor openshift installations of Che use it